app.js

```diff
@@ -1,19 +1,33 @@
 const express = require('express');
 const {
     getTopics,
-    getApi
+    getApi,
+    getComments
 } = require('./Controllers/controllers');
 
 const app = express();
 
 
 app.get("/api/topics", getTopics)
 app.get("/api", getApi)
+app.get("/api/articles/:article_id/comments", getComments)
 
 app.all("/*", (request, response) => {
     response.status(404).send({message: "Not found"})
 })
 
+app.use((err, req, res, next) => {
+    if(err.code === "22P02") {
+        res.status(400).send({message: "Bad request"})
+    }
+    next(err);
+})
+app.use((err, req, res, next) => {
+    if(err.code === 404) {
+        res.status(404).send({message: "Match not found"})
+    }
+    next(err);
```

Comment:
These `next(err)` calls should be in `else` blocks otherwise you're going to pass the same error to your 500 handler.